examples/workflows/artifact_with_fanout.py

```diff
@@ -1,22 +1,23 @@
 from hera.workflows import DAG, Artifact, Workflow, script
+from hera.workflows.archive import NoneArchiveStrategy
```

Comment:
Is this importable as `from hera.workflows import NoneArchiveStrategy`?

Reply:
Yes it is! Fixing